### ISSUE TITLE
Document Example With Vapor 4

### ISSUE THREAD

**Issue by eloit** [sersoft-gmbh/swift-smtp#5]
Hello,

I've been trying to implement this package in a Vapor 4 app without success.   I believe I am correctly important the packages and are able to create Configuration structs.   From what I see in the code,  the framework creates its own EventLoopGroup, and creates a default mailer.   

For the life of me I can't seem to get this to initialize properly.   Clearly I am missing something that is the most obvious.

Could you provide a example of how you implement in Vapor.  How to initialize in Configure.swift. (if that is appropriate).  I'm pretty sure once initialized I can get it to work,  but if you can have a simple example in a route where a mail is sent (similar to the CLI example) this would be so great.

Thanks again.

**Comment by ffried** [comment]
@eloit Thanks for the heads-up here. I've just extended our README. Let me know if that helps!

**Comment by eloit** [comment]
@ffried Thanks so much.  I was able to get it running easily.  The logTransmissions feature was essential for me to get it working with my server.    At first I was getting 503 Errors.   This is due to the fact that the server that I am connecting to is a ESMTP.   This initial handshake of "HELO" resulted in it responding but not accepting commands (Hostgator).  I could not figure out how to override this so I ended up forking the code and changed the initial handshake to "EHLO" and it connected and transacted flawlessly.   

Very helpful module that gets me out of a bind.  Thanks again!
